AltBeacon.Library/src/AltBeacon/Beacon/BeaconParser.cs

```diff
@@ -416,8 +417,7 @@ public BeaconParser SetBeaconLayout(string beaconLayout)
 
                 if (!found)
                 {
-                    ////TODO LogManager
-                    ////LogManager.d(Tag, "cannot parse term " + term);
+                    Logger.Debug("cannot parse term " + term);
```

Comment:
Btw, I can write add `LogManager.VerboseLoggingEnabled` condition to every logger so that they wont log anything like it supposed to do and also if I use logger with params like `Logger.Debug("cannot parse term {0}", term);`, log string won't even get built (but of course in this case it will change nothing since there is only one param).

Also, this is a debug log not a verbose that is why I asked to change name of the flag since `VerboseLoggingEnabled` kind of misleading for disabling debug logs.